### DISCUSSION TITLE
Real-time Notifications for Changes in Files Managed by Chezmoi

### DISCUSSION THREAD

**Discussion by 8ta4** [twpayne/chezmoi#3086] [Q&A]
I often run `chezmoi status` and discover that some files managed by Chezmoi have been altered. The issue is, I don't always know what caused these changes or when they were made.

Who is messing with my `~`? Who is gaslighting me chez moi?

I need to find out the truth. I need to get notifications whenever a file changes, so I can catch the culprit and confront them. I'm thinking of using `fswatch`. But maybe there's a better way.

Do you know of any feature in Chezmoi that can help me with this? Or any community solution that has worked for you?

**Comment by twpayne** [comment]
I haven't used `fswatch`, there is also [`watchman`](https://facebook.github.io/watchman).

A combination of `fswatch`/`watchman` with [`chezmoi managed`](https://www.chezmoi.io/reference/commands/managed/) should work.
